src/core/ActionMenu/ActionMenuDivider/ActionMenuDivider.baseStyles.tsx

```diff
@@ -0,0 +1,19 @@
+import { css } from 'styled-components';
+import { SuomifiTheme } from '../../theme';
+
+export const baseStyles = (theme: SuomifiTheme) => css`
+  &.fi-action-menu-divider {
+    height: 17px;
+    padding-left: 15px;
```

Comment:
Should these 15px be with spacing tokens?

Reply:
Fixed